haskell/defs.bzl

```
@@ -356,6 +352,20 @@ def haskell_binary(
         **repl_kwargs
     )
 
+def make_repl_kwargs(args_list, kwargs):
```

```suggestion
def make_repl_kwargs(args_list, kwargs):
    """Create extra attributes for the auto-generated haskell_repl target.
    
    Copies the extra attributes specified in `args_list` from the extra
    `haskell_library|binary|test` attributes listed in `kwargs`.
    
    Adds a `manual` tag so that the REPL is not built by default.
    """
```
Adding a doc-string.